src/spineml/simulator/inputValue.cc

```diff
@@ -342,7 +350,21 @@ std::unique_ptr<SpineMLSimulator::InputValue::Base> SpineMLSimulator::InputValue
         return std::unique_ptr<Base>(new TimeVaryingArray(dt, numNeurons, node));
```

Comment:
I'd say use ``make_unique`` instead, but GeNN still supports C++11, right?

Reply:
Yeah, GeNN does indeed use C++11. For whatever reason Visual Studio 2013 is the minimum targeted compiler (it was because it was installed on university managed Windows machines but, now it's more that my windows machine at home has that...) and that barely does C++11.

Reply:
Ok cool. There is always the option of copying and pasting a ``make_unique`` function from the internet -- it is handy (and safer in certain situations)